docs/reference/migration/migrate_8_0.asciidoc

```diff
@@ -34,6 +34,7 @@ coming[8.0.0]
 * <<breaking_80_snapshots_changes>>
 * <<breaking_80_threadpool_changes>>
 * <<breaking_80_transport_changes>>
+* <<breaking_80_java_time_changes>>
```

Comment:
We already have a mapping changes section. I wonder if this would be a better fit there than it's own section.

Reply:
will move to a mapping changes